be/src/exprs/time_functions.cpp

```diff
@@ -982,24 +983,34 @@ StatusOr<ColumnPtr> TimeFunctions::to_unix_from_datetime(FunctionContext* contex
             result.append_null();
         } else {
             timestamp = timestamp < 0 ? 0 : timestamp;
-            timestamp = timestamp > INT_MAX ? 0 : timestamp;
+            timestamp = timestamp > MAX_UNIX_TIMESTAMP ? 0 : timestamp;
             result.append(timestamp);
         }
     }
 
     return result.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> TimeFunctions::to_unix_from_datetime(FunctionContext* context, const Columns& columns) {
+    int func_version = context->state()->func_version();
+    if (func_version > 4) {
```

Comment:
magic number. any comment to describe it?

Reply:
no, usually there is no comment for specific function version. BUt I can add comment since this func version. 

Usually when we increase function verison, it's because we want to introduce some feature that is not incompatible to old BE.